ouroboros-consensus/src/Ouroboros/Consensus/NodeKernel.hs

```diff
@@ -279,25 +278,45 @@ forkBlockForging IS{..} blockForging =
 
         trace $ TraceLedgerState currentSlot bcPrevPoint
 
-        -- We require the ticked ledger view in order to construct the ticked
-        -- 'ChainDepState'.
-        ledgerView <-
-          case runExcept $ forecastFor
-                           (ledgerViewForecastAt
-                              (configLedger cfg)
-                              (ledgerState unticked))
-                           currentSlot of
-            Left err -> do
-              -- There are so many empty slots between the tip of our chain and
-              -- the current slot that we cannot get an ledger view anymore
-              -- In principle, this is no problem; we can still produce a block
-              -- (we use the ticked ledger state). However, we probably don't
-              -- /want/ to produce a block in this case; we are most likely
-              -- missing a blocks on our chain.
-              trace $ TraceNoLedgerView currentSlot err
-              exitEarly
-            Right lv ->
-              return lv
+        -- Tick the ledger state for the 'SlotNo' we're producing a block for
```

Comment:
```suggestion
        -- Tick the ledger state that corresponds to the 'SlotNo' we're producing a block for
```

Reply:
This suggestion is wrong. We are "ticking /a ledger state/ for /a slot/" as opposed to "ticking the ledger state at a slot"